packages/docs/docs/ssi/trust-model.md

```diff
@@ -4,16 +4,17 @@ sidebar_position: 4
 
 # SSI Trust Model
 
-SSI completely changes the paradigm of online data sharing and brings it closer to the physical world. There are three entities in the **VC trust model:**
+SSI completely changes the paradigm of online data sharing and brings it closer to the physical world. There are three entities in the **VC trust model**:
 
-- **Issuer** that issues the credential
-- **Holder** that is the owner and subject of the credential
-- **Verifier** that receives and verifies the credential
+- **the issuer** that issues the credential
+- **the holder** that is the owner and subject of the credential
```

Comment:
I would remove 'that' after each entity. Example:
- **the holder** is the owner and subject of the credential